retirejs-zap-plugin/pom.xml

```diff
@@ -32,6 +32,9 @@
                 <directory>src/main/resources-filtered</directory>
                 <filtering>true</filtering>
             </resource>
+            <resource>
+                <directory>src/main/javahelp</directory>
```

Comment:
I'm not sure we want to support javahelp resource.
Is this something that will be show in ZAP UI?

Reply:
I was not familiar with javahelp. It seems to be a deprecated project : https://github.com/javaee/javahelp

Reply:
It is shown within ZAP. This is the standard setup for all existing addons.

Reply:
It's also used to generate content for zaproxy.org (https://www.zaproxy.org/docs/desktop/addons/)

Reply:
It is recommended for all ZAP add-ons - its how our users expect to find info about add-ons and we also use it to generate the online content.

Reply:
And it is linked from the marketplace page: https://www.zaproxy.org/addons/